### PR TITLE
feat(tutorial): complete step 5

### PR DESCRIPTION
**Pull request by jeancarl**
Completed the fifth part of the tutorial: https://www.carbondesignsystem.com/tutorial/react-step-5

### CODE REVIEW

**Comment by jeancarl** [comment]
http://carbon-tutorial-jbisson.mybluemix.net/

File: src/index.scss [hunk 1]
```diff
@@ -1,29 +1,16 @@
...
-    padding-left: 1rem;
-    padding-right: 1rem;
-  }
-}
-
-/// Padding was introduced in 10.3.0, needs an option to let grid set the viewport gutter
-/// @link https://github.com/carbon-design-system/carbon/issues/3010
-.bx--content {
-  padding: 0;
-}
```
Comment (kellychurchill) [inline src/index.scss]: These should not have been removed. Their removal is resulting in too much padding in your layout: http://carbon-tutorial-jbisson.mybluemix.net/#/

File: src/index.scss [hunk 2]
```diff
@@ -1,29 +1,16 @@
-$feature-flags: (
```
[the remark this answers is not in the document]
Comment (jeancarl) [reply src/index.scss]: Fixed

File: src/index.scss (hunk 1 above)
Comment (jeancarl) [reply src/index.scss]: Fixed, please check again.

**Pull request closed without merging**